metrix-simulator/src/metrix2assess.cpp

```diff
@@ -742,11 +747,14 @@ int Calculer::metrix2Assess(const std::shared_ptr<Variante>& var, const vector<d
                             if (config::inputConfiguration().useAllOutputs()
                                 || (config::configuration().displayResultatsRedispatch()
                                     && fabs(val) >= EPSILON_SORTIES)) {
-                                fprintf(fr,
-                                        ("R2B ;%d;%s;" + PREC_FLOAT + ";\n").c_str(),
-                                        incidentsContraignants.find(icdt)->second,
-                                        s.c_str(),
-                                        val);
+                                tuple<const int, const string, const double> tupleToAdd(incidentsContraignants.find(icdt)->second, s, val);
```

Comment:
`s` variable is not explicit. 
Could be :
```suggestion
                                tuple<const int, const string, const double> tupleToAdd(incidentsContraignants.find(icdt)->second, grp->nom_, val);
```
or :
```suggestion
                                tuple<const int, const string, const double> tupleToAdd(incidentsContraignants.find(icdt)->second, grpName, val);
```
with grpName -> `string grpName  = grp->nom_;` instead of `string s = grp->nom_;`